config/routes.rb

```diff
@@ -2,6 +2,14 @@
   root 'sauces#index'
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
+  
   resources :sauces, only: [:index, :new, :create]
+
+  get "/sauces/:id", to: "sauces#show"
```

Comment:
this should go to `"homes#index"`. Otherwise, you would just be re-using a controller. 

Let it be known: what you have done IS viable. But we are asking students to use the pattern I showed last week on the API clinic

Reply:
per this, you should delete the created ERB file

Reply:
changed this route to point at "homes#index" and deleted "/views/sauces/show.html.erb"